zulip_bots/setup.py

```diff
@@ -56,7 +56,7 @@
         "lxml",
         "BeautifulSoup4",
         "typing_extensions>=4.5.0",
-        'importlib-metadata >= 3.6; python_version  < "3.10"',
+        "importlib-metadata>=3.6",
```

Comment:
Fix for #829, included here as it is necessary for the Py 3.12 tests (but not for the 3.11 one).

Reply:
Hmm, isn't the point of #829 that we can just drop this dependency and use the built-in module in 3.10? Or do we need to wait until we drop 3.9 support to drop it?

Reply:
Yes, we can't drop it before dropping 3.9 support.